packages/near-membrane-dom/src/window.ts

```diff
@@ -238,3 +238,22 @@ export function tameDOM(
     env.remap(blueRefs.WindowPropertiesProto, WindowPropertiesDescriptors);
     env.remap(blueRefs.WindowProto, blueRefs.WindowProtoDescriptors);
 }
+
+export function linkUnforgeables(
+    env: VirtualEnvironment,
+    blueGlobalThis: Window & typeof globalThis
+) {
+    // The test of instance of event target is important to discard environments in which
+    // a fake window (e.g. jest) is not following the specs, and can break this
+    // membrane.
+    if (blueGlobalThis.EventTarget && blueGlobalThis instanceof EventTarget) {
```

Comment:
Well, you're right... in 0.7.0 we moved the logic into the `init`, making it generic, therefore the protection was needed. In this PR we moved back to `window.ts`, which is only supposed to be used in a real browser. Prior to 0.7, those environments were loading the nodejs version of the lib, meaning they never rewire the unforgeables assuming there were none. But then yesterday, when debugging, I noticed that then running jest tests in the repo, both, the nodejs and the browser version of the libraries are loaded, not sure why, so I decided to keep it around even though they are in the window.ts, which is mean to run in a qualifying environment with a valid window instance.

Reply:
>  But then yesterday, when debugging, I noticed that then running jest tests in the repo, both, the nodejs and the browser version of the libraries are loaded

In near-membrane repo? I wonder if this because the jest config uses `'<rootDir>/packages'`? I will take a look

Reply:
Jest uses `jsdom` which simulates a browser environment so I can see it needing the guard.